airdrop/job/eligibility.py

```diff
@@ -0,0 +1,95 @@
+
+import sys
+sys.path.append('/opt')
+import time
+import uuid
+
+from airdrop.job.repository import Repository
+from common.exception_handler import exception_handler
+from common.utils import generate_lambda_response
+from airdrop.config import BALANCE_DB_CONFIG, NETWORK, SLACK_HOOK
+from common.logger import get_logger
+from pydoc import locate
+from decimal import Decimal
+
+logger = get_logger(__name__)
+
+class EligibilityProcessor:
+    def __init__(self):
+        self._airdrop_db = Repository(NETWORK["db"])
```

Comment:
Are we sure that we want to keep the DB connection at the object level?

Reply:
Yes, this is a batch job that runs on a schedule. At a later point we will remove the DB connection for Balances and switch this to Alembic